.gitignore

```diff
@@ -5,3 +5,4 @@ target
 **/src/api/*
 !**/src/api/mod.rs
 vmm/common/Cargo.lock
+!/vmm/sandbox/src/bin/
```

Comment:
Is it helpful to replace Line2 `bin` with `/bin`

Reply:
Thanks for the advice. I made the changes.

`/bin` ignores all `bin` directories in the project directory.